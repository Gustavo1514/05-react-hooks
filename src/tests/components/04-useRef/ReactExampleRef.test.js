const { shallow } = require('enzyme')
const {
  default: MultipleCustomHooks
} = require('../../../components/03-examples/MultipleCustomHooks')

describe('Pruebas sobre <ReactExampleRef/>', () => {
  test('debe mostrarse correctamente', () => {
    const wrapper = shallow(<MultipleCustomHooks />)
    expect(wrapper).toMatchSnapshot()
  })

  test('debe mostrar el componente < MultipleCustomHooks/>', () => {})
})
