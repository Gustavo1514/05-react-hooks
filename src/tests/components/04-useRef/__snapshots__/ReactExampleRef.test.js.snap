// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre <ReactExampleRef/> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Breaing Bad Cuotes
  </h1>
  <hr />
  <div
    className="alert alert-info text-center"
  >
    Loading
  </div>
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Nueva Frase
  </button>
</div>
`;
